Cornelica/scenario/013_1.txt

```diff
@@ -56,16 +56,16 @@
 
 @bs f=リリィ@微笑 x=350 y=50 t=400
 
-[Lily] "Moving onto " Magical power ", this is what is necessary for Al to move"
+[Lily]"Moving onto \C[2]Magical power\C[0], \nthis is what is necessary for you to move"
 
 @bs f=リリィ@哀しみ x=350 y=50 t=200
 
-[Lily] "Get anywhere near a Succubus, you'll suck up the Charm, and your magical power will decrease, so be careful."
+[Lily]"Get anywhere near a Succubus, you'll suck up the \C[2]Charm\C[0], \nand your \C[2]Magical power\C[0] will decrease, so be careful."
```

Comment:
Maybe it should be `they'll suck up your Charm`? Think that got lost in translation slightly.

Reply:
The sentence means that if you get closer the charm goes up... I pharse like you will absorb charm and the magic power goes down......

Reply:
Maybe its just how I read it then.